src/pool/srv_iv.c

```diff
@@ -715,24 +693,23 @@ pool_iv_map_ent_update(d_sg_list_t *dst_sgl, struct pool_iv_entry *src_iv)
 }
 
 static int
-pool_iv_prop_ent_copy(struct pool_iv_entry *dst_iv,
-		      struct pool_iv_entry *src_iv)
+pool_iv_prop_ent_copy(struct pool_iv_entry *dst_iv, struct pool_iv_entry *src_iv)
 {
-	daos_prop_t	*prop_fetch;
-	int		rc = 0;
+	daos_prop_t *prop_fetch;
+	int          rc;
 
 	prop_fetch = daos_prop_alloc(DAOS_PROP_PO_NUM);
 	if (prop_fetch == NULL)
 		return -DER_NOMEM;
 
 	rc = pool_iv_prop_g2l(&src_iv->piv_prop, prop_fetch);
 	if (rc) {
-		daos_prop_free(prop_fetch);
-		D_ERROR("prop g2l failed: rc %d\n", rc);
-		return rc;
+		D_ERROR("pool_iv_prop_g2l failed: rc " DF_RC "\n", DP_RC(rc));
```

Comment:
Should this be
```suggestion
		D_ERROR("pool_iv_prop_g2l failed: " DF_RC "\n", DP_RC(rc));
```

Reply:
Yes, that would be nicer.  Will re-push on Monday.

Reply:
Done.